### DISCUSSION TITLE
Which settings would generate the built firmware?

### DISCUSSION THREAD

**Discussion by dhysuiej** [mriscoc/Ender3V2S1#1174] [Q&A] [answered]
Thanks for this firmware, I've been using it for a while and it's a really awesome.

I decided to try building the firmware myself to see if enabling `SMOOTH_ENCODER_MENUITEMS` would solve an issue with my printer's knob jumping values all over the place.

I used the settings bellow to build the firmware, the printer is an Ender 3 v2 with CRTouch.
![Screenshot 2023-12-13 at 1 08 42 PM](https://github.com/mriscoc/Ender3V2S1/assets/82825136/7d3089b3-831e-43d4-8418-9019526e0713)

These are the generated files.
[MyConfiguration20231209.zip](https://github.com/mriscoc/Ender3V2S1/files/13663723/MyConfiguration20231209.zip)

All I did was to uncomment the '//' in the line with `SMOOTH_ENCODER_MENUITEMS` and copy the `.h` files to the Marlin folder, and the file `platformio.ini` to the root folder, in the default branch of Ender3V2S1 cloned repository.

However when I try the option for manual tramming, before trying to build the mesh automatically, the head goes to the center of the bed and probes that point, then moves a couple centimeters to the front and probes again. Then  no matter which corner I select in the manual tramming menu the head just probes the bed in the same spot and doesn't move anywhere else. This issue behavior is not present in the pre-built firmware. So I wanted to see if some could share the exact files (or settings) used in the pre-build firmware **Ender3V2-422-BLTUBL-MPC**. It would be very much appreciated.

**Comment by mriscoc** [comment, marked as answer]
Check your mesh inset https://github.com/mriscoc/Ender3V2S1/wiki/Calibration-Guides

**Reply by dhysuiej** [reply]
Thanks a lot 🙂
